src/mapml/handlers/ContextMenu.js

```diff
@@ -723,13 +775,157 @@ export var ContextMenu = L.Handler.extend({
+          //edge case at index 0
+          if (!this.isRunned && this.activeIndex === 0 && !this._items[this.activeIndex].el.el.disabled){
+            this._setActiveItem(0);
+            this.isRunned = true;
+          } else { //edge case over
+            let nextIndex = this.activeIndex + 1;
+            while (this._items[nextIndex].el.el.disabled) {
+              nextIndex++;
+              if (nextIndex >= this._items.length) {
+                nextIndex = 0;
+              }
+            }
+            this._setActiveItem(nextIndex);
+          }
+        } else {
+          let nextIndex = 0;
+          while (this._items[nextIndex].el.el.disabled) {
+            nextIndex++;
+            if (nextIndex >= this._items.length) {
+              nextIndex = 0;
+            }
+          }
+          this._setActiveItem(nextIndex);
+        }
+      }
+    } else if (key === 39) { //right arrow
+      if (document.activeElement.shadowRoot === null) { //do nothing when null
+      } else {
+        if (document.activeElement.shadowRoot.activeElement.innerHTML === this._items[5].el.el.innerHTML && //'copy'
+          this._coordMenu.hasAttribute('hidden')){
+          this._showCoordMenu();
+          this._coordMenu.children[0].focus();
+        } else if (document.activeElement.shadowRoot.activeElement.innerHTML === this._items[5].el.el.innerHTML && 
+          !this._coordMenu.hasAttribute('hidden')) {
+          this._coordMenu.children[0].focus();
+        }
+      }
+    } else if (key === 37) { //left arrow
+      if (!this._coordMenu.hasAttribute('hidden') && 
+          document.activeElement.shadowRoot !== null) {
+        if (document.activeElement.shadowRoot.activeElement.innerHTML === this._coordMenu.children[0].innerHTML ||
+        document.activeElement.shadowRoot.activeElement.innerHTML === this._coordMenu.children[1].innerHTML || 
+        document.activeElement.shadowRoot.activeElement.innerHTML === this._coordMenu.children[2].innerHTML){
+          this._coordMenu.setAttribute('hidden','');
+          this._setActiveItem(5);
+        }
+      }
+    } else if (key === 27) { //esc key
+      if (document.activeElement.shadowRoot === null) {
+        this._hide();
+      } else {
+        if (!this._coordMenu.hasAttribute('hidden')) {
+          if (document.activeElement.shadowRoot.activeElement.innerHTML === this._coordMenu.children[0].innerHTML ||
+          document.activeElement.shadowRoot.activeElement.innerHTML === this._coordMenu.children[1].innerHTML || 
+          document.activeElement.shadowRoot.activeElement.innerHTML === this._coordMenu.children[2].innerHTML){
+            this._coordMenu.setAttribute('hidden','');
+            this._setActiveItem(5);
+          }
+        } else {
+          this._hide();
+        }
+      }
+    } else if (e.shiftKey && key === 9){
+      let arr = ['Back (<kbd>Alt+Left Arrow</kbd>)', 'Forward (<kbd>Alt+Right Arrow</kbd>)', 'Reload (<kbd>Ctrl+R</kbd>)',
```

Comment:
Looks like the localization is ignored here.  Let's undo these changes and log the tab issue as to be completed in the future, no need to rush it.